lib/racecar/config.rb

```diff
@@ -223,7 +236,8 @@ def load_consumer_class(consumer_class)
         consumer_class.name.gsub(/[a-z][A-Z]/) { |str| "#{str[0]}-#{str[1]}" }.downcase,
       ].compact.join
 
-      self.parallel_workers = consumer_class.parallel_workers
+      self.forks = consumer_class.forks
+      self.threads = consumer_class.threads
```

Comment:
I think this should be configurable with the config itself as well; could we add a new config key and have this be e.g. `self.threads = consumer_class.threads || self.threads`?